timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -62,66 +62,72 @@ def get_default_hps(key, prediction_length):
     context_length = max(prediction_length * 2, MINIMUM_CONTEXT_LENGTH)
     default_model_hps = {
         "toy": {
+            "ETS": {
+                "maxiter": 20,
+                "seasonal": None,
+            },
+            "ARIMA": {
+                "maxiter": 10,
+                "order": (1, 0, 0),
+            },
+            "DeepAR": {
+                "epochs": 10,
+                "num_batches_per_epoch": 10,
+                "context_length": 5,
+            },
             "SimpleFeedForward": {
                 "epochs": 10,
                 "num_batches_per_epoch": 10,
                 "context_length": 5,
             },
-            "Transformer": {"epochs": 10, "num_batches_per_epoch": 10, "context_length": 5},
-            "DeepAR": {"epochs": 10, "num_batches_per_epoch": 10, "context_length": 5},
-            "ETS": {"maxiter": 20, "seasonal": None},
-            "ARIMA": {
-                "maxiter": 10,
-                "order": (1, 0, 0),
-                "seasonal_order": (0, 0, 0),
+            "Transformer": {
+                "epochs": 10,
+                "num_batches_per_epoch": 10,
+                "context_length": 5,
             },
         },
         "default": {
-            "ETS": {
-                "maxiter": 200,
-                "trend": "add",
-                "seasonal": "add",
-            },
-            "ARIMA": {
-                "maxiter": 50,
-                "order": (1, 1, 1),
-                "seasonal_order": (0, 0, 0),
-            },
+            "ETS": {},
+            "ARIMA": {},
             "SimpleFeedForward": {
                 "context_length": context_length,
             },
-            "Transformer": {
+            "DeepAR": {
                 "context_length": context_length,
             },
-            "DeepAR": {
+            "TemporalFusionTransformer": {
                 "context_length": context_length,
             },
         },
         "default_hpo": {
```

Comment:
The search space should be somewhat stable, just want to double click on it so we don't change it soon.